NomaiVR/Hands/Hand.cs

```diff
@@ -1,58 +1,218 @@
+                    new CustomSkeletonHelper.Retargetable(sourceWristTransform.Find(FingerBoneName("index", 1)), targetWristTransform.Find(FingerBoneName("index", 1))), //Proximal
+                    new CustomSkeletonHelper.Retargetable(sourceWristTransform.Find(FingerBoneName("index", 2)), targetWristTransform.Find(FingerBoneName("index", 2))), //Middle
+                    new CustomSkeletonHelper.Retargetable(sourceWristTransform.Find(FingerBoneName("index", 3)), targetWristTransform.Find(FingerBoneName("index", 3))), //Distal
+                    prefabTransform.Find(BoneToTarget("finger_index_r_aux", true)) //aux
+                ),
+                new CustomSkeletonHelper.Finger(
+                    new CustomSkeletonHelper.Retargetable(sourceWristTransform.Find(FingerBoneName("ring", 0)), targetWristTransform.Find(FingerBoneName("ring", 0))), //Metacarpal
+                    new CustomSkeletonHelper.Retargetable(sourceWristTransform.Find(FingerBoneName("ring", 1)), targetWristTransform.Find(FingerBoneName("ring", 1))), //Proximal
+                    new CustomSkeletonHelper.Retargetable(sourceWristTransform.Find(FingerBoneName("ring", 2)), targetWristTransform.Find(FingerBoneName("ring", 2))), //Middle
+                    new CustomSkeletonHelper.Retargetable(sourceWristTransform.Find(FingerBoneName("ring", 3)), targetWristTransform.Find(FingerBoneName("ring", 3))), //Distal
+                    prefabTransform.Find(BoneToTarget("finger_ring_r_aux", true)) //aux
+                ),
+            };
+
+
+            var skeletonPoser = skeletonDriver.fallbackPoser;
+            skeletonPoser.skeletonMainPose = fallbackRelax;
+            skeletonPoser.skeletonAdditionalPoses.Add(fallbackPoint);
+            skeletonPoser.skeletonAdditionalPoses.Add(fallbackFist);
+
+            //Point Fallback
+            skeletonPoser.blendingBehaviours.Add(new SteamVR_Skeleton_Poser.PoseBlendingBehaviour()
+            {
+                action_bool = SteamVR_Actions.default_Grip,
+                enabled = true,
+                influence = 1,
+                name = "point",
+                pose = 1,
+                value = 0,
+                type = SteamVR_Skeleton_Poser.PoseBlendingBehaviour.BlenderTypes.BooleanAction,
+                previewEnabled = true,
+                smoothingSpeed = 16
+            });
+
+            //Fist Fallback
+            skeletonPoser.blendingBehaviours.Add(new SteamVR_Skeleton_Poser.PoseBlendingBehaviour()
+            {
+                action_bool = SteamVR_Actions.default_GrabPinch,
+                enabled = true,
+                influence = 1,
+                name = "fist",
+                pose = 2,
+                value = 0,
+                type = SteamVR_Skeleton_Poser.PoseBlendingBehaviour.BlenderTypes.BooleanAction,
+                previewEnabled = true,
+                smoothingSpeed = 16
+            });
```

Comment:
My interpretation of this is that, if there's no skeleton available, it will use the specified buttons to define these poses. I tested removing the skeleton definitions from my personal SteamVR input bindings, but pressing Grip didn't seem to trigger the "point" gesture as I expected. Am I misinterpreting this? The hands looked fine with the skeleton disabled though, so it's absolutely fine if that's what's supposed to happen too.